app/controllers/admin/api/services_controller.rb

```diff
@@ -48,7 +48,7 @@ def index
   def create
     service = current_account.services.build
     create_service = ServiceCreator.new(service: service)
-    create_service.call!(service_params)
+    create_service.call(service_params)
     service.reload if service.persisted? # It has been touched
```

Comment:
I think that this line is not needed anymore... It was there for the way that it was creating the service back then. I remember that there was a test failing if I originally removed that line back then, so u should try now and see what happens :smile:

Reply:
Yes I remember this one :)

Reply:
@hallelujah I pushed the change. should I remove?

Reply:
Confirmed: we still need the line :joy: